Philips.CodeAnalysis.Common/Philips.CodeAnalysis.Common.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>net7.0;netstandard2.0</TargetFrameworks>
-    <LangVersion>latest</LangVersion>
+    <LangVersion>preview</LangVersion>

```

Comment:
preview?

Reply:
Upgraded to .NET 8 and reverted the `<LangVersion>`